store/slice/obj.test.ts

```diff
@@ -0,0 +1,73 @@
+import { asserts, describe, it } from "../../test.ts";
+import { configureStore, updateStore } from "../../store/mod.ts";
+
+import { createObj } from "./obj.ts";
+const tests = describe("createObj()");
+
+export type TCurrentUser = {
```

Comment:
Any reason why you decided to use `type` here?  I generally stick to `interface`.